app/controllers/rents_controller.rb

```diff
@@ -1,5 +1,6 @@
 class RentsController < ApiController
   before_action :authenticate_user!
+  before_action :set_locale
```

Comment:
it's better if you move to the api controller. It's something that you will need in some other controllers